web/src/components/GridCell.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import Typography from '@material-ui/core/Typography'
+import ExpandIcon from '@material-ui/icons/ExpandMore'
+import IconButton from '@material-ui/core/IconButton'
+import SelectionDialog from './SelectionDialog'
+
+import TimePassed from '../assets/time_passed.png'
+import SlackIcon from '../assets/slack_icon.png'
+import DockerIcon from '../assets/docker_icon.png'
+import StatusChip from './StatusChip';
+import SlackInfoDialog from './SlackInfoDialog';
+import ImageInfoDialog from './ImageInfoDialog';
+
+const imgAlt = "not found"
```

Comment:
Not sure what this is for, we can just omit the alt tag from images because we're not trying to be accessible.

Reply:
Agreed. The tag is only present so the linter passes and so there is no compiler warnings.